### PR TITLE
fix: Change order consignment outlet position from REPLACE to AFTER

### PR DESCRIPTION
**Pull request by surya-prakash-singh1**
ix: Change order consignment outlet position from REPLACE to AFTER

### CODE REVIEW

**Review by JedidiahAwuku — DISMISSED** [review]

File: package-lock.json [hunk 1]
```diff
@@ -20647,7 +20647,7 @@
     },
     "projects/eslint": {
       "name": "@spartacus/eslint-plugin-eslint",
-      "version": "0.0.0-dev",
+      "version": "6.0.0-1",
```
Comment (WeizhengSap) [inline package-lock.json]: why we need change this?

File: feature-libs/order/styles/components/order-details/_order-details-items.scss [hunk 2]
```diff
@@ -35,12 +35,6 @@
     padding: 0 30px;
   }
 
-  .cx-review-header {
```
Comment (WeizhengSap) [inline feature-libs/order/styles/components/order-details/_order-details-items.scss:38]: I think we need this. please double check

Comment (JedidiahAwuku) [reply feature-libs/order/styles/components/order-details/_order-details-items.scss:38]: Hi Wei, it seems styling for the `cx-review-header` has been already made in the `_review.scss` file or is this deliberate? The UI looks the same when added and removed.

File: package-lock.json (hunk 1 above)
Comment (JedidiahAwuku) [reply package-lock.json]: Removed. thanks

**Review by WeizhengSap — APPROVED** [review]

**Pull request merged**